ee/hogai/graph/agent_modes/compaction_manager.py

```diff
@@ -101,7 +104,13 @@ async def calculate_token_count(
         return await self._get_token_count(model, messages, tools, **kwargs)
 
     def update_window(
-        self, messages: Sequence[T], summary_message: ContextMessage, start_id: str | None = None
+        self,
+        messages: Sequence[T],
+        summary_message: ContextMessage,
+        agent_mode: AgentMode,
+        start_id: str | None = None,
+        # Delete when modes are rolled out
```

Comment:
`nit`: add explicit TODO so a bit easier to spot


```suggestion
        # TODO: Delete when modes are rolled out
```